Add new metrics collected by debuginfo (#62)
dgraph-io/dgraph-docs: content/howto/retrieving-debug-information.md

## Profiling Information with `debuginfo`

> Instead of sending a request to the server for each CPU, Memory, goroutine profile, and you can use the `debuginfo` command to collect all the profiles along with several metric you need in one go.

Reviewer: ```suggestion
Instead of sending a request to the server for each CPU, Memory, and goroutine profile, you can use the `debuginfo` command to collect all the profiles along with several metrics you need in one go.
```

I0217 03:13:04.445784   12495 debugging.go:104] please wait... (30s)
I0217 03:13:04.446058   12495 debugging.go:92] saving jemalloc metric in /tmp/dgraph-debuginfo695159006/alpha_jemalloc.gz
I0217 03:13:04.446073   12495 debugging.go:102] fetching information over HTTP from http://localhost:8080/state
I0217 03:13:04.446086   12495 debugging.go:104] please wait... (30s)
I0217 03:13:04.447259   12495 debugging.go:92] saving state metric in /tmp/dgraph-debuginfo695159006/alpha_state.gz
I0217 03:13:04.447270   12495 debugging.go:102] fetching information over HTTP from http://localhost:8080/health
I0217 03:13:04.447277   12495 debugging.go:104] please wait... (30s)
I0217 03:13:04.447788   12495 debugging.go:92] saving health metric in /tmp/dgraph-debuginfo695159006/alpha_health.gz
I0217 03:13:04.452051   12495 run.go:122] Debuginfo archive successful: dgraph-debuginfo695159006.tar.gz
```

When the command finishes, `debuginfo` returns the tarball's file name. If no destination has been specified, the file will be created in the same directory from where you ran the `debuginfo` command. 

> The followings are the information that have been collected by the `debuginfo` command:

Reviewer: ```suggestion
The following files are the metrics that have been collected by the `debuginfo` command:
```

## Profiling Information with `debuginfo`

> Instead of sending a request to the server for each CPU, Memory, and goroutine profile, you can use the `debuginfo` command to collect all the profiles along with several metrics you need in one go.

Reviewer 2: ```suggestion
Instead of sending a request to the server for each CPU, memory, and `goroutine` profile, you can use the `debuginfo` command to collect all of these profiles, along with several metrics.
```

Author: done

  -x, --archive            Whether to archive the generated report (default true)
  -d, --directory string   Directory to write the debug info into.
  -h, --help               help for debuginfo

>   -m, --metrics strings    List of metrics & profile to dump in the report. (default [heap,cpu,state,health,jemalloc,trace,metrics,vars,trace,goroutine,block,mutex,threadcreate])

Reviewer 2: ```suggestion
  -m, --metrics strings    List of metrics & profiles to dump in the report. (default: [heap,cpu,state,health,jemalloc,trace,metrics,vars,trace,goroutine,block,mutex,threadcreate])
```

Author: done

  -s, --seconds uint32     Duration for time-based metric collection. (default 30)
  -z, --zero string        Address of running dgraph zero.
```

#### The metrics flag (`-m`)

By default, `debuginfo` collects:
- `heap`
- `cpu`
- `state`
- `health`
- `jemalloc`
- `trace`
- `metrics`
- `vars`
- `trace`
- `goroutine`
- `block`
- `mutex`
- `threadcreate`

> If needed, you can collect some of them (not necessarily all). For example, this command will collect only `jemalloc` and `health` profiles:

Reviewer 2: ```suggestion
If you want to collect some, but not all, of the profiles, you can use a command like the following example that collects only the `jemalloc` and `health` profiles:
```

Author: the seconds flag has been removed
